roles/config-rh-sso/tasks/setup-rh-sso.yml

```diff
@@ -3,11 +3,47 @@
   import_role:
     name: ../roles/config-packages
 
-- name: 'Update SSO bind addresses'
-  replace:
-    path: /etc/opt/rh/rh-sso7/keycloak/standalone/standalone.xml
-    regexp: '127.0.0.1'
-    replace: '0.0.0.0'
+- name: 'Update Red Hat SSO management/public interface IP address'
+  shell: >
+    "/opt/rh/rh-sso7/root/usr/share/keycloak/bin/jboss-cli.sh" -c
+    --command='/interface=management:write-attribute(name=inet-address,value="${jboss.bind.address:0.0.0.0}")'
+  shell: >
+    "/opt/rh/rh-sso7/root/usr/share/keycloak/bin/jboss-cli.sh" -c
+    --command='/interface=public:write-attribute(name=inet-address,value="${jboss.bind.address:0.0.0.0}")'
+  args:
+    executable: "/bin/bash"
+
+- name: 'Create a keystore to store SSL certificates'
+  java_keystore:
+    name: example
+    certificate: "{{lookup('file', 'path/to/the/file.cer') }}"
+    private_key: "{{lookup('file', 'path/to/the/file.key') }}"
+    password: "{{ sso_keystore_pass }}"
+    dest: /etc/security/keystore.jks
```

Comment:
Please use a default variable for this to avoid discrepancies below - make sure to update all places consuming this string to use the variable

Reply:
Added the `sso_keystore_path` variable.